lib/util.js

```diff
@@ -191,59 +192,47 @@ function getLink(obj, linkName, altName) {
   } else {
     return link;
   }
-}
+};
+
+util.getNativeConsole = function() {
+  return window.console;
+};
 
-function warn(text) {
+util.getConsole = function() {
+  var nativeConsole = util.getNativeConsole();
```

Comment:
I'll make sure this works for @nikhilvenkatraman-okta before merging. I don't think it throws an error, I just don't think console exists.